docs/source/cluster_management/sweep/sweep-endpoints.rst

```diff
@@ -5,24 +5,20 @@ AtlasDB Sweep Endpoints
- +                                          +-------------------------------------------------+                                                                                                  |
- |                                          | ``startRow``: base16 encoded start row          |                                                                                                  |
- +                                          +-------------------------------------------------+                                                                                                  |
- |                                          | ``maxCellTsPairsToExamine``: cells to examine   |                                                                                                  |
- +                                          +-------------------------------------------------+                                                                                                  |
- |                                          | ``candidateBatchSize``: cells to read per batch |                                                                                                  |
- +                                          +-------------------------------------------------+                                                                                                  |
- |                                          | ``deleteBatchSize``: cells to delete per batch  |                                                                                                  |
- +------------------------------------------+-------------------------------------------------+--------------------------------------------------------------------------------------------------+
+ +-----------------------+-------------------------------------------------+---------------------------+---------------------------------------------------------------------------------+
+ | Endpoint              | Query Parameters                                | Required/Optional         | Description                                                                	 |
+ +=======================+=================================================+===========================+=================================================================================+
+ |``/sweep/sweep-table`` | ``tablename``: fully qualified table name       | Required                  |Sweep a particular table from EMPTY startRow with default ``SweepBatchConfig``.  |
+ +                       +-------------------------------------------------+---------------------------+                                                                                 |
+ |                       | ``startRow``: base16 encoded start row          | Optional                  |                                                                                 |
```

Comment:
We should probably add that the default is empty, which means that it will start from the first row.